src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,344 @@
+
++ An **InMigration** is the migration of an Individual into the area of study. This can come in two flavors: Internal or External. 
+
+ - An Internal InMigration is when an Individual migrates from one place within the study area to another place still in the study area. 
+
+ - An External InMigration is when an Individual comes from outside of the area of study and migrates to a location within it. 
+
++ An **OutMigration** is the migration of an Individual to outside of the area of study. 
+ 
+Examples of migrations in general would be an individual who accepts a new job outside of the area of study and moves to be closer to it, an individual marries another individual and moves from one place to another within the study area, etc.
+
++ Pregnancies are complex multi-stage events in the real world and so in the OpenHDS are modeled differently than the other Update Events. A pregnancy has three discrete event entities: **PregnancyObservation**, **PregnancyOutcome**, **PregnancyResult**. 
+ 
+ - A PregnancyObservation is the initial record of the pregnancy by a FieldWorker. 
+
+ - The PregnancyOutcome is a record of what the outcome of the pregnancy was, this includes information about actual delivery date and the number and type of births the pregnancy yielded. 
+ 
+ - Finally, the PregnancyResult is a record for each birth from a particular PregnancyOutcome.
+
++ Lastly, a **Death** is the death of an individual and the last way an individual can leave an area of study.
+
+Just as with the Census and Relation entities, all of these entities are collected by and dependent on a FieldWorker entity. However, these Update events are inherently different than the census information: they’re time dependent. The census data is time independent demographic and geographic information collected from the study area. However, as time moves forward the population that the census data was initially gathered from will slowly change as people come, go, are born, or die. In order to capture this data the same population must be visited periodically so that the Update events can be accounted for within the OpenHDS. This act of visiting the population is modeled explicitly within the OpenHDS with an entity called a **Visit**. All of the Update event entities are dependent on a Visit. However, a FieldWorker can resurvey a group within the study area and create a visit, but find that nothing has changed thus leaving the visit empty.

```

Comment:
I think Visit should have its won **heading** and intro, rather than introducing it in the middle of a paragraph.